StyleCop.Analyzers/StyleCop.Analyzers.Test/ReadabilityRules/SA1130UnitTests.cs

```diff
@@ -1010,5 +983,48 @@ public void Test()
 
             await VerifyCSharpFixAsync(testCode, DiagnosticResult.EmptyDiagnosticResults, fixedCode, CancellationToken.None).ConfigureAwait(false);
         }
+
+        protected virtual DiagnosticResult[] GetExpectedResultCodeFixSpecialCases()
+        {
+            return new[]
+            {
+                Diagnostic().WithLocation(8, 20),
+                Diagnostic().WithLocation(9, 20),
+                Diagnostic().WithLocation(10, 25),
+                Diagnostic().WithLocation(11, 30),
+                Diagnostic().WithLocation(12, 30),
+                Diagnostic(CS1065).WithLocation(12, 53),
+                Diagnostic().WithLocation(13, 30),
+                Diagnostic(CS7014).WithLocation(13, 47),
+                Diagnostic().WithLocation(14, 30),
+                Diagnostic(CS1670).WithLocation(14, 47),
+                Diagnostic().WithLocation(15, 25),
+                Diagnostic(CS1669).WithLocation(15, 42),
```

Comment:
💡 Let's move all the `Diagnostic()` items back into the test itself, and only have the `CS*` diagnostics in this virtual method.

Reply:
Sure. I thought of that too briefly, but ended up not doing it. 👍

Reply:
Added them as annotations in the tested code instead. This removed the need for two methods before and after the fix. Ok simplification?